champtc/opencti
# AB#5816 Keycloak Admin Client

Adding Keycloak admin calls to getting user information.

## Testing
* Checkout this branch
* Do `yarn install` in the `opencti-graphql` folder
* Start the backend
* Start the frontend
* Go to localhost client and login
* No errors should occur

Should see normal user information as before the difference is in the code when the user information is retrieved.
